CodeGen/include/Luau/IrData.h

```diff
@@ -59,7 +59,8 @@ enum class IrCmd : uint8_t
 
     // Load a TValue from memory
     // A: Rn or Kn or pointer (TValue)
-    // B: int (optional 'A' pointer offset)
+    // B: int/none (optional 'A' pointer offset)
```

Comment:
Great, now we have to update all instructions with optional arguments! :)

Reply:
😅 fwiw we use undef/tag in another place, but LOAD_TVALUE already used none/int, which is why I annotated the tag (and the int for consistency). I am also assuming "none" is exactly equivalent to "0", which is why I specify 0 instead of "none" so that we don't have gaps in IR op arguments.